nullaway/src/main/java/com/uber/nullaway/ErrorBuilder.java

```diff
@@ -302,15 +307,49 @@ boolean symbolHasSuppressWarningsAnnotation(Symbol symbol, String suppression) {
     return false;
   }
 
-  static String errMsgForInitializer(Set<Element> uninitFields) {
-    String message = "initializer method does not guarantee @NonNull ";
+  static int getLineNumForElement(Element uninitField, VisitorState state) {
+    Tree fieldTree = getTreesInstance(state).getTree(uninitField);
+    if (fieldTree == null)
+      throw new RuntimeException(
+          "When getting the line number for uninitialized field, can't get the fieldTree from the element.");
+    DiagnosticPosition position =
+        (DiagnosticPosition)
+            fieldTree; // Expect Tree to be JCTree and thus implement DiagnosticPosition
+    TreePath path = state.getPath();
+    JCCompilationUnit compilation = (JCCompilationUnit) path.getCompilationUnit();
+    JavaFileObject file = compilation.getSourceFile();
+    DiagnosticSource source = new DiagnosticSource(file, null);
+    return source.getLineNumber(position.getStartPosition());
+  }
+
+  // Generate the message for uninitialized fields, including the line number for fields.
+  static String errMsgForInitializer(Set<Element> uninitFields, VisitorState state) {
+    StringBuilder message = new StringBuilder("initializer method does not guarantee @NonNull ");
+    Element uninitField;
     if (uninitFields.size() == 1) {
-      message += "field " + uninitFields.iterator().next().toString() + " is initialized";
+      uninitField = uninitFields.iterator().next();
+      message.append("field ");
+      message.append(uninitField.toString());
+      message.append("(Line ");
```

Comment:
Sorry to be picky, but can we have a space before the `(` and a lowercase `l`?  So it will read, e.g., `field f (line 33)`

Reply:
It's good to be picky as it makes the code better! I will adjust the message!